test/sdk-benchmarks/src/main/java/software/amazon/awssdk/benchmark/utils/BenchmarkUtil.java

```diff
@@ -15,21 +15,35 @@
 
 package software.amazon.awssdk.benchmark.utils;
 
+import io.netty.handler.ssl.SslContext;
+import io.netty.handler.ssl.SslProvider;
 import java.net.URI;
 import java.time.Instant;
 import java.time.LocalDateTime;
 import java.time.ZoneOffset;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import org.openjdk.jmh.infra.Blackhole;
 import software.amazon.awssdk.core.SdkBytes;
 import software.amazon.awssdk.services.protocolec2.model.AllTypesRequest;
+import software.amazon.awssdk.utils.Logger;
 
 /**
  * Contains utility methods and constants used by the benchmarks
  */
 @SuppressWarnings("unchecked")
 public final class BenchmarkUtil {
 
-    public static final int PORT_NUMBER = 8089;
-    public static final URI LOCAL_URI = URI.create(String.format("http://localhost:%s", PORT_NUMBER));
+    public static final String DEFAULT_JDK_SSL_PROVIDER = "jdk";
+    public static final String OPEN_SSL_PROVIDER = "openssl";
+
+    public static final int HTTP_PORT_NUMBER = 8089;
+    public static final int HTTPS_PORT_NUMBER = 9999;
+    public static final int CONCURRENT_CALLS = 50;
```

Comment:
Any reason why 50?

Reply:
Keeping it 50 because the default max connection is 50. Don't think we need to use a larger value for regression tests. What do you think?